### PR TITLE
Bootstrapping bundle init

### PR DESCRIPTION
**Pull request by PrimalPimmy**

### CODE REVIEW

**Comment by shivaccuknox** [comment]
Please attach  the call flow diagram to the PR so that reviewers will have some context

File: controllers/pkg/reconcilers/spire-bootstrap/reconciler.go [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+	// 	}
+	// 	if found {
+	// 		// speeds up the loop
+	// 		break
+	// 	}
+	// }
+
+	// // Example: Update the status if necessary
+
+	socketPath := "unix:///spiffe-workload-api/agent.sock"
```
Comment (shivaccuknox) [inline controllers/pkg/reconcilers/spire-bootstrap/reconciler.go]: Create a config map for socket path, Vault IP address in nephio-system ns.

File: controllers/pkg/reconcilers/spire-bootstrap/reconciler.go [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+	jwtSource, err := workloadapi.NewJWTSource(ctx, clientOptions)
+	if err != nil {
+		log.Info("Unable to create JWTSource: %v", err)
+	}
+	defer jwtSource.Close()
+
+	audience := "TESTING"
+	spiffeID := spiffeid.RequireFromString("spiffe://example.org/nephio")
+
+	jwtSVID, err := jwtSource.FetchJWTSVID(ctx, jwtsvid.Params{
```
Comment (shivaccuknox) [inline controllers/pkg/reconcilers/spire-bootstrap/reconciler.go]: Add a function to get a jwt token (for current workload)

File: controllers/pkg/reconcilers/spire-bootstrap/reconciler.go [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+	config.Address = vaultAddr
+	client, err := vault.NewClient(config)
+	if err != nil {
+		log.Error(err, "Unable to create Vault client:")
+	}
+
+	// Set the client token
+	client.SetToken(clientToken)
+
+	// Retrieve the secret
```
Comment (shivaccuknox) [inline controllers/pkg/reconcilers/spire-bootstrap/reconciler.go]: Create a function to retrieve secrets. Also, create a function to write library

**Comment by PrimalPimmy** [comment]
Ref: https://github.com/5GSEC/SIG-Security/issues/9

**Comment by PrimalPimmy** [comment]
TODOs:
Create configmap/env variables for things like Vault IP, Spire unix domain socket, etc

**Review by shivaccuknox — APPROVED** [review]
There are some changes need to this commit. Will raise separate issues for tracking them

**Pull request merged**